subsys/dm/Kconfig

```diff
@@ -15,32 +15,41 @@ if DM_MODULE
 
 menu "nRF DM (Distance Measurement)"
 
-choice NRF_DM_TIMER
-	default NRF_DM_TIMER2
+choice DM_TIMER
+	default DM_TIMER2
 	prompt "Timer to use for the nrf_dm library."
 
-config NRF_DM_TIMER0
+config DM_TIMER0
```

Comment:
not sure if timer0 should be a valid option, since that is used by MPSL/SDC.

Reply:
Agree, even if other protocol is used for synchronization, the MPSL timeslot API is used for accessing Radio, so TIMER0 should probably be excluded.

Reply:
As far as I know when a Timeslot is granted the application has access to TIMER0.
So if only the `nrf_dm_proc_execute` function (from the nrf_dm library) uses a timer, it should not be a problem. Because this function is called in the Timeslot.

Reply:
ah right. I think you're right @maje-emb 👍